assets/flox.zdotdir/.zlogout

```diff
@@ -0,0 +1,10 @@
+zlogout="${FLOX_ORIG_ZDOTDIR:-$HOME}/.zlogout"
```

Comment:
```suggestion
# Source "${FLOX_ORIG_ZDOTDIR:-$HOME}/.zlogin" and /etc/zlogin if they exist
#
# This script are called through `ZDOTDIR=<.../flox.zdotdir> zsh --no-globalrcs`,
# thus flox has to source the appropriate rc files itself.
# The value of $ZDOTDIR set before calling `flox activate`
# is preserved as `$FLOX_ORIG_ZDOTDIR`

zlogout="${FLOX_ORIG_ZDOTDIR:-$HOME}/.zlogout"
```

Reply:
Done in 2c86b1a88e853180afead2a3d7c87541f743cb3c